esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -28,6 +28,8 @@ for more details on the boards see mlrs-wifi-bridge-boards.h
 - M5Stack M5Stamp C3U Mate
   board: ESP32C3 Dev Module
   ATTENTION: when the 5V pin is used, one MUST not also use the USB port, since they are connected internally!!
+- M5Stack ATOM Lite
+  board: M5Stack ATOM Lite
```

Comment:
this should be exactly what you have to choose in Arduino IDE, which - IMHO - is `M5Stack-ATOM`

Reply:
OK, changed.